adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -270,7 +270,7 @@ class ADAMContextSuite extends ADAMFunSuite {
   sparkTest("can read a small .vcf file") {
     val path = resourcePath("small.vcf")
 
-    val vcs = sc.loadGenotypes(path).toVariantContext.collect.sortBy(_.position)
+    val vcs = sc.loadGenotypes(path).toVariantContextRDD.rdd.collect.sortBy(_.position)
```

Comment:
Since we have this
```scala
object ADAMContext {
  implicit def genomicRDDToRDD[T](gRdd: GenomicRDD[T]): RDD[T] = gRdd.rdd
```
https://github.com/bigdatagenomics/adam/blob/master/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala#L114

why is the ```.rdd``` here and elsewhere necessary?  Implicits work fine except for when they don't?